packages/world/ts/data.ts

```diff
@@ -0,0 +1,10 @@
+// Follows https://viem.sh/docs/actions/wallet/signTypedData#usage
+
+export const types = {
```

Comment:
A constant called `types` (as a global export from the package) is way to generic of a name here. This is the type of the message that is used as payload for `registerDelegationWithSignature`? We should reflect that in the variable name. My preference would be to call the constant `DelegationSignature` and use it as `{ DelegationSignature }` in `signTypedData`. I think we should also rename the file from `data.ts` to something more specific.

Reply:
Agreed! Was just being lazy and copying the naming from the viem example